front/app/utils/dateUtils.ts

```diff
@@ -126,6 +146,14 @@ export function convertSecondsToDDHHMM(seconds: number) {
   return `${formattedDaysLeft}:${formattedHoursLeft}:${formattedMinutesLeft}`;
 }
 
+export function capitalizeDates(locale: Locale) {
+  const localesWithCapitalizedWeekdays = ['en', 'de'];
+  if (localesWithCapitalizedWeekdays.includes(locale)) {
```

Comment:
This one I'm a bit unsure about. When we have "Monday, February 4th" for example, most languages will write "monday, february 4th". From what I can find, only English and German capitalize weekday names :thinking:  So I added this util to decide whether a weekday name should be upper or lowercase (since the "moment" library doesn't include any helpers for this). :thinking: If you have another idea though I'm definitely open to suggestions!